Extensions/Today/TodayViewController.swift

```diff
@@ -300,41 +258,35 @@ class ButtonWithSublabel: UIButton {
     }
 
     private func performLayout() {
-        self.snp_removeConstraints()
-
         let titleLabel = self.label
         titleLabel.textColor = UIColor.whiteColor()
 
         self.titleLabel?.removeFromSuperview()
         addSubview(titleLabel)
 
         let imageView = self.imageView!
-
         let subtitleLabel = self.subtitleLabel
         subtitleLabel.textColor = UIColor.lightGrayColor()
         self.addSubview(subtitleLabel)
 
         imageView.snp_remakeConstraints { make in
             make.centerY.equalTo(self.snp_centerY)
-            make.right.equalTo(titleLabel.snp_left).offset(-TodayUX.horizontalWidgetMargin)
+            make.left.equalTo(self.snp_left)
```

Comment:
Prefer `leading`/`trailing` over `left`/`right` for RTL compatibility.

Also, you can chain these together if you want (`make.leading.centerY`).

(Same comments for other constraint definitions below.)